app/themes/default/views/layouts/default.html.slim

```diff
@@ -0,0 +1,43 @@
+doctype html
+html
+  head
+    title
+      - if content_for?(:page_title)
+        = "#{content_for(:page_title)} - "
+      | Coursemology2
+    meta http-equiv="X-UA-Compatible" content="IE=edge"
+    = viewport_meta_tag
+    = application_resources
+    = stylesheet_link_tag 'default/all', media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag 'default/all', defer: true, 'data-turbolinks-track' => true
+    = csrf_meta_tags
+
+  body
+    nav.navbar.navbar-inverse.navbar-fixed-top role="navigation"
+      div.container
+        div.navbar-header
+          button.navbar-toggle.collapsed type="button" data-toggle="collapse" data-target="#site-navigation-navbar" aria-expanded="false" aria-controls="navbar"
+            span.sr-only Toggle navigation
+            span.icon-bar
+            span.icon-bar
+            span.icon-bar
+          a.navbar-brand href="#" Coursemology
+        div.collapse.navbar-collapse#site-navigation-navbar
+          ul.nav.navbar-nav.pull-right
+            li.active
+              a href="#" Courses
+            li
+              a href="#about" Help
+            - if user_signed_in?
+              li
+                = link_to('Logout', destroy_user_session_path, method: :delete)
+            - else
+              li
+                = link_to('Sign Up', new_user_registration_path)
+              li
+                = link_to('Login', new_user_session_path)
+
+    div.container-fluid
+      = flash_messages
+      div class=page_class
```

Comment:
this is correct. I want to call the `page_class` helper and set the class of the div to that. equivalent to (in erb):

```erb
<div class="<%= page_class %>">...
```